### PR TITLE
Span to measure the part of db writes that is after the merge/extraction

### PR DESCRIPTION
**Pull request by dureuill**

### CODE REVIEW

File: crates/milli/src/update/new/channel.rs [hunk 1]
```diff
@@ -17,8 +17,8 @@ use crate::vector::Embedding;
 use crate::{DocumentId, Index};
 
 /// The capacity of the channel is currently in number of messages.
-pub fn extractor_writer_channel(cap: usize) -> (ExtractorSender, WriterReceiver) {
-    let (sender, receiver) = crossbeam_channel::bounded(cap);
+pub fn extractor_writer_channel(_cap: usize) -> (ExtractorSender, WriterReceiver) {
+    let (sender, receiver) = crossbeam_channel::unbounded();
```
Comment (Kerollmops) [inline crates/milli/src/update/new/channel.rs:21]: Can we rollback this, please? I am touching this part in my PR with BBQueues and it removes the memory control we were working on.

**Review by Kerollmops — CHANGES_REQUESTED** [review]

File: crates/milli/src/update/new/channel.rs (hunk 1 above)
Comment (dureuill) [reply crates/milli/src/update/new/channel.rs]: removed

**Review by Kerollmops — APPROVED** [review]
Nice, thank you! Very useful span 🐬
bors merge

**Comment by curquiza** [comment]
bors merge

**Pull request merged**